guide/designing-products/user-research.md

```diff
@@ -171,7 +173,7 @@ Links to:
    layout = "float-left"
 %}
 
-User research should not be a one-sided extraction of information but a collaboration between researchers and users to find better solutions. Since researchers initiate research efforts, it is up to them to create frameworks that respect users' privacy. That means full upfront disclosure, open collaboration, and a mutual understanding of how research results will be used.
+User research should not be a one-sided extraction of collaboration between researchers and users. Since researchers initiate research efforts, it is up to them to create frameworks that respect users’ privacy. That means upfront disclosures, open collaboration, and transparency about how research results are used.
```

Comment:
```suggestion
User research should not be a one-sided extraction of information, but a collaboration between researchers and users. Since researchers initiate research efforts, it is up to them to create frameworks that respect users’ privacy. That means upfront disclosures, open collaboration, and transparency about how research results are used.
```
Fixed broken sentence.